Makefile

```diff
@@ -18,22 +23,22 @@ audit:
 .PHONY: build
 build:
 	@mkdir -p $(BUILD_ARCH)/$(BIN_DIR)
-	go build -o $(BUILD_ARCH)/$(BIN_DIR)/$(MAIN) cmd/dp-search-api/main.go
+	go build $(LDFLAGS) -o $(BUILD_ARCH)/$(BIN_DIR)/$(MAIN) cmd/dp-search-api/main.go
 
 .PHONY: debug
 debug: build
-	HUMAN_LOG=1 go run -race cmd/dp-search-api/main.go
+	HUMAN_LOG=1 go run $(LDFLAGS) -race cmd/$(MAIN)/main.go
 
 .PHONY: acceptance-publishing
 acceptance-publishing: build
```

Comment:
`acceptance-publishing` and `acceptance-web` make targets can be removed. These were in here for legacy acceptance testing; as you can see both these targets are the same as `debug`. So no value keeping these